roles/get_image_hash/tasks/get_image_hash.yml

```diff
@@ -1,8 +1,16 @@
-- name: "Get {{ item.key }} image hash"
-  shell: "skopeo inspect --authfile {{ local_pull_secret_path }} docker://{{ item.value.image }}:{{ item.value.tag }}"
-  register: result
-  changed_when: false
+- name: "A hash wasn't provided, inspect the image to figure it out"
+  when: not (item.value.hash | default('')) != ''
```

Comment:
I like this. I've been meaning to cache the hashes for a while but haven't gotten round to it this makes it pretty trivial.